src/preferences/dialog/dlgprefreplaygaindlg.ui

```diff
@@ -246,6 +221,43 @@
     </widget>
    </item>
    <item>
+    <widget class="QGroupBox" name="groupBoxAnalysis">
+     <property name="title">
+      <string>Analysis</string>
+     </property>
+     <layout class="QVBoxLayout" name="verticalLayout_3">
+      <item>
+       <widget class="QRadioButton" name="radioButtonRG2">
+        <property name="text">
+         <string>ReplayGain 2.0 (EBU R 128)</string>
```

Comment:
Yes. I would like to keep the "EBU R 128" sting though since the target loudness (reference) is set able in Mixxx anyway. 
Do you have an idea to improve this?